File: src/main/Application.java
package main;

import java.util.ArrayList;
import java.util.List;

import institution.University;
import institution.interlink.Internship;
import person.Student;

public class Application {
    public static void main(String[] args) {
        University university  = new University("CH.U.I.");
        
        university.addStudent(new Student("Andrew Kostenko", 59));
        university.addStudent(new Student("Julia Veselkina", 35));
        university.addStudent(new Student("Maria Perecrest", 66));
        university.addStudent(new Student("Peter Petrov", 29));
        university.addStudent(new Student("Oleg Urbanovich", 96));
        university.addStudent(new Student("Sam Samsonovich", 74));
        university.addStudent(new Student("John Johnsonich", 87));

        Internship internship = new Internship("Interlink", university.getAverageKnowledge());
        for(int i=0; i<university.studentsList.size(); i++) {
			internship.setStudent(university.studentsList.get(i));
        }
        System.out.println("Average knowledge of students is:  " +  university.averageKnowledge);
        System.out.println("List of internship's students:");
        System.out.println(internship.getStudents());
        
        
    }
}

File: src/main/Application.js
var University = require('../institution/University');
var Internship = require('../institution/interlink/Internship');
var Student = require('../person/Student');
var fs = require('fs');
var file = fs.readFileSync('../students.txt').toString().split('\r\n');
var studentsArray = [];

for (let line of file){
    studentsArray.push(line.split(','));
}


var university = new University("CH.U.I.");
for (let student of studentsArray){
    university.addStudent(new Student(student[0], student[1]));
}


var internship = new Internship("Interlink", university.getAverage());
    for(let student of university.studentsList){
        internship.setStudent(student);
    }
    console.log(`\nAverage knowledge of students is ${Math.round(university.averageKnowledge)}`)
    console.log("List of internship's students:");
    console.log(internship.getStudents());
